browser/ui/webui/brave_webui_source.cc

```diff
@@ -1059,6 +1059,11 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "confirmationNotice",
             IDS_BRAVE_WEBCOMPATREPORTER_CONFIRMATION_NOTICE },
       }
+    }, {
+      std::string("brave_webtorrent"), {
+        { "webtorrentPeer", IDS_WEBTORRENT_PEER },
+        { "webtorrentPeers", IDS_WEBTORRENT_PEERS },
+      }
```

Comment:
Where are these used? As far as I know, this area map is used for WebUI pages only and not extension. If it is needed, whilst this is probably ok, I think this technique (putting all the strings for all WebUIs in a single file (which is very long now) is deprecated. And would prefer if it went directly in the specific WebUI.